spec/signalfx-transform-helper-spec.js

```diff
@@ -1,81 +1,44 @@
 const signalfxHelper = require('../signalfx-transform-helper');
-const sinon = require('sinon');
 const chai = require('chai');
 const expect = chai.expect;
 
 describe('signalfx-transform-helper', () => {
 
-  it('should leave flat objects as they are', () => {
-    const obj = { 'key1': 'value', 'key2': 5};
-    const flattened = signalfxHelper.toKeyValueMap(obj);
-    expect(flattened).to.deep.equal(obj);
-  });
-
-  it('should sanitize field names to include only a-zA-Z0-9-_', () => {
-    const obj = { 'IP Address': "123.45.67.89", 'field': { 'nested/object:name': 'test'}};
-    const flattened = signalfxHelper.toKeyValueMap(obj);
-    expect(flattened.IP_Address).to.equal("123.45.67.89");
-    expect(flattened.field_nested_object_name).to.equal('test');
+  it('should sanitize keys', () => {
+    const obj = {'!ke@£y $%^1&': 'value'};
```

Comment:
I think that the original test ('it should sanitize field names to include only a-zA-Z0-9_') should stay, as it had examples that were inspired by true CloudWatch events. With that in mind, I also think that "_" is better as a separator. In your example, it difficult to see that the original name of the field and the one after sanitization are referring to the same field.